### PR TITLE
Fixed formatting & styling of copyright/contributors

### LINKED ISSUE #8: Update intro screen
I initially started [TinyFlowerBeds](https://twitter.com/tinyflowerbeds) in order to better understand Python's useful (and silly) applications and how to create apps that will interact with social media. After I took a simple approach for it, I made an intro screen that shows the name of the project, a copyright notice for [the project's license](LICENSE) and **also added my name with a copyright sign**.

https://github.com/AlwaysLivid/TinyFlowerBeds/blob/ad525983d4a143db1f34802e38ea2460c55bab4d/src/bot.py#L18

I essentially abandoned the project, but I've revived it, because I noticed that it's a simple fun thing to work on and that others could learn some useful stuff by fiddling around with it (just as I did). When I started working on it, however, I did not expect anybody else to improve the thing that I did. There has been some interest revolving around it and there have been people that have contributed their own source code! According to the [GNU General Public License v3](LICENSE), the rights to the source code that they wrote belongs to them. I'm not the only copyright holder anymore!

So, here's what should change;
* The year should be updated from `2018-2019` to `2018-2020`.
* Change my alias (`AlwaysLivid`) to something that includes my name, as well as the `contributors`, as well as a link that lists all of the people that have contributed to the bot on GitHub.
* Make sure that the alignment remains intact. Feel free to make changes all over the string in order to maintain that.

### PR DESCRIPTION
**Pull request by GobindB**
Updates the intro screen for the bot. Made changes to the string which displays copyright and contributor details.

Fixes #8

### CODE REVIEW

File: src/bot.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 @description: Bot that posts a tiny flower bed on Twitter every few hours.
 '''
 
-import tweepy
+# import tweepy
```
Comment (n0toose) [inline src/bot.py:9]: Why are you commenting out the `tweepy` module?

File: src/bot.py [hunk 2]
```diff
@@ -15,7 +15,9 @@
 print("""
                        TinyFlowerBeds
 
-            Copyright (C) 2018-2019 AlwaysLivid
+Copyright (C) 2018-2020 AlwaysLivid, Shankhanil, karl01101101
```
Comment (n0toose) [inline src/bot.py:18]: Could you make the text align towards the center? Additionally, could you change the text to `TinyFlowerBeds contributors` and then point people to this repository instead?

The text is a bit unreasonably long.

**Review by n0toose — CHANGES_REQUESTED** [review]

File: src/bot.py (hunk 1 above)
Comment (GobindB) [reply src/bot.py:9]: My bad forgot to uncomment `tweepy`, will resolve.

File: src/bot.py (hunk 2 above)
Comment (GobindB) [reply src/bot.py:18]: Sure

**Comment by n0toose** [comment]
The pull request seems dormant. Closing.

**Pull request closed without merging**